tests/selfReportToIWF.test.ts

```diff
@@ -0,0 +1,171 @@
+import { ServerlessCallback } from '@twilio-labs/serverless-runtime-types/types';
+import axios from 'axios';
+import { omit } from 'lodash';
+import {
+  handler as selfReportToIWF,
+  Body,
+  IWFSelfReportPayload,
+} from '../functions/selfReportToIWF';
+
+import helpers, { MockedResponse } from './helpers';
+
+jest.mock('@tech-matters/serverless-helpers', () => ({
+  ...jest.requireActual('@tech-matters/serverless-helpers'),
+  functionValidator: (handlerFn: any) => handlerFn,
+}));
+
+jest.mock('axios');
+
+const baseContext = {
+  getTwilioClient: (): any => ({}),
+  DOMAIN_NAME: 'serverless',
+  IWF_API_CASE_URL: 'IWF_API_CASE_URL',
+  IWF_REPORT_URL: 'IWF_REPORT_URL',
+  IWF_SECRET_KEY: 'IWF_SECRET_KEY',
+  PATH: 'PATH',
+  SERVICE_SID: undefined,
+  ENVIRONMENT_SID: undefined,
+};
+
+const defaultPayload = {
+  secret_key: 'secret_key',
+  case_number: 'case_number',
+  user_age_range: '<13',
+};
+
+describe('selfReportToIWF', () => {
+  beforeAll(() => {
+    const runtime = new helpers.MockRuntime({});
+    // eslint-disable-next-line no-underscore-dangle
+    helpers.setup({}, runtime);
+  });
+  afterAll(() => {
+    helpers.teardown();
+  });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('Should return status 400', async () => {
```

Comment:
Can you name the test to specify the circumstances under which it should return a 400? Same for all the other tests

Reply:
It's done.